pom.xml

```diff
@@ -412,6 +412,7 @@
         <module>api/metrics-api-common</module>
         <module>api/metrics-api-jaxrs</module>
         <module>api/metrics-api-jaxrs-1.1</module>
+        <module>hawkular-component</module>
```

Comment:
What do you think about moving this module under api? Since it is meant for external consumption?